HW2/HW2_4/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+
+namespace HW2_4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int a1 = 12;
+            Int32 a2 = 13;
+            Console.WriteLine(a1);
```

Comment:
we need to display the type of the variable - a1.GetType() method should be used.